extensions/wikia/Answers/AttributionCache.class.php

```diff
@@ -250,79 +277,83 @@ private function sortContribs(Array $contribs) {
 	 * @param int $userId user id
 	 * @param int $pageId page id for calculating edits per article
 	 */
-	private function rebuildUserEditPoints($userId, $pageId = 0) {
+	private function rebuildUserEditPoints( $userId, $pageId = 0 ) {
 		global $wgAnswerHelperIDs, $wgMemc;
-		wfProfileIn(__METHOD__);
+		wfProfileIn( __METHOD__ );
 
 		$dbs = wfGetDB( DB_MASTER );
 		$tables = array( 'revision' );
 		$conditions = array( 'rev_user' => $userId );
-		if(!empty($pageId)) {
+		if ( !empty( $pageId ) ) {
 			$tables[] = 'page';
 			$conditions[] = 'page_id=rev_page';
 			$conditions['page_id'] = $pageId;
 		}
 		/* @TODO FIXME: respect your DB resources, never count on MASTER */
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)

[project-id]: wikia:app
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app